modules/openapi-generator/src/main/resources/dart2/api_helper.mustache

```diff
@@ -4,16 +4,16 @@ class QueryParam {
   const QueryParam(this.name, this.value);
 
   final String name;
-  final String value;
+  final String? value;
```

Comment:
can query have name but no value? and for the line below, it has to be empty string not `null` string, as end user won't be able to distinguish intentional `null` string from no value. I think both fields should be not-null in this class